src/views/MyProfile/Components/VictoryPromiseDisplay/index.js

```diff
@@ -223,118 +228,114 @@ export default class VictoryPromiseDisplay extends React.Component {
+          <Grid>
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Christian Character</Typography>
+                  {
+                    'Opportunities encouraging faith formation and its connection to living, learning and leading with others'
+                  }
+                </React.Fragment>
+              }
+            >
+              {IMG_CC}
+            </HoverText>
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Intellectual Maturity</Typography>
+                  {
+                    'Opportunities to extend critical reasoning, deepen understanding, and ignite imagination'
+                  }
+                </React.Fragment>
+              }
+            >
+              {IMG_IM}
+            </HoverText>
           </Grid>
-        </section>
+          <Grid container justify="center">
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Lives of Service</Typography>
+                  {
+                    "Opportunities to lend one's strengths and talents with our partners to our neighbors"
+                  }
+                </React.Fragment>
+              }
+            >
+              {IMG_LS}
+            </HoverText>
+            <HoverText
+              title={
+                <React.Fragment>
+                  <Typography color="inherit">Leadership Worldwide</Typography>
+                  {
+                    "Opportunities to develop one's understanding and influence in God's amazing, dynamic and challenging world"
+                  }
+                </React.Fragment>
+              }
+            >
+              {IMG_LW}
+            </HoverText>
+          </Grid>
+        </Grid>
       );
     } else {
       content = (
-        <section>
-          <Grid container>
-            <Grid item>
-              <Card>
-                <CardContent>
-                  <Grid container direction="row" alignItems="center">
-                    <Grid item xs={7}>
-                      <CardHeader title="Victory Promise" />
-                    </Grid>
-                    <Grid item xs={5} align="right">
-                      <Button
-                        variant="contained"
-                        style={style.button}
-                        onClick={() => this.changeMode()}
-                      >
-                        Change Style
-                      </Button>
-                    </Grid>
+        <Grid container justify="center">
+          <Polar
+            className="victory-promise"
+            data={{ labels: this.state.labels, datasets: this.state.datasets }}
+            options={this.state.options}
+          />
+        </Grid>
+      );
+    }
+
+    return (
+      <section>
+        <Grid container>
```

Comment:
Do you want spacing between the items in this Grid?

Reply:
I don't think so. It looked fine to me without it